chacra/models/binaries.py

```diff
@@ -28,6 +29,9 @@ class Binary(Base):
     project_id = Column(Integer, ForeignKey('projects.id'))
     project = relationship('Project', backref=backref('binaries', lazy='dynamic'))
 
+    repo_id = Column(Integer, ForeignKey('repos.id'))
```

Comment:
This needs to be a many to many relationship, right?  Binaries can be in multiple repos and repos can have multiple binaries.

http://docs.sqlalchemy.org/en/improve_toc/orm/basic_relationships.html#many-to-many